coremltools/converters/mil/frontend/torch/internal_graph.py

```diff
@@ -229,6 +229,14 @@ def get_arguments(alist):
                     args.append(get_arguments(i))
                 elif isinstance(i, (int, float)):
                     args.append(i)
+                # This is necessitated by backward compatibility:
+                # * TorchScript used to store dtype as integers
+                # * Subsequently, we built our PyTorch converter based on numbered dtypes
+                # * Now EXIR uses dtype directly...
+                # * Until refactoring our converter for EXIR,
```

Comment:
Can you give a little more details about the necessary refactoring? Can we only do this refactoring after we no longer support TorchScript?